tests/e2e/DropDownFilter/DropDownFilter.spec.ts

```diff
@@ -23,4 +33,29 @@ describe("Drop down filter", () => {
         const itemValue = defaultFilter.listViewFirstItem.getHTML();
         expect(itemValue).toContain(textFilterValue);
     });
+    it("should filter when used multiple times", () => {
+        browser.url("/p/index");
+
+        const dropDownSort = $(".widget-drop-down-sort .form-control");
+        dropDownSort.click();
+        dropDownSort.$$("option")[0].click();
+        const itemValueSortFirst = page.listViewFirstItem.getHTML();
+        expect(itemValueSortFirst).toContain("Belgium");
+
+        dropDownSort.click();
+        dropDownSort.$$("option")[1].click();
+        const itemValueSortSecond = page.listViewFirstItem.getHTML();
+        expect(itemValueSortSecond).toContain("United Kingdom");
+
+        const dropDownFilter = $(".widget-drop-down-filter .form-control");
+        dropDownFilter.click();
+        dropDownFilter.$$("option")[1].click();
```

Comment:
What value are we clicking here? 😄

Reply:
English countries and All countries. The results are countries, official language and continent.